src/pipx/main.py

```diff
@@ -908,8 +920,10 @@ def _add_environment(subparsers: argparse._SubParsersAction, shared_parser: argp
             variables and platform specific default values.
 
             Available variables:
-            PIPX_HOME, PIPX_BIN_DIR, PIPX_MAN_DIR, PIPX_SHARED_LIBS, PIPX_LOCAL_VENVS,
-            PIPX_LOG_DIR, PIPX_TRASH_DIR, PIPX_VENV_CACHEDIR, PIPX_DEFAULT_PYTHON, USE_EMOJI, PIPX_HOME_ALLOW_SPACE
+            PIPX_HOME, PIPX_GLOBAL_HOME, PIPX_BIN_DIR, PIPX_GLOBAL_BIN_DIR,
+            PIPX_MAN_DIR, PIPX_GLOBAL_MAN_DIR, PIPX_SHARED_LIBS, PIPX_LOCAL_VENVS,
+            PIPX_LOG_DIR, PIPX_TRASH_DIR, PIPX_VENV_CACHEDIR, PIPX_DEFAULT_PYTHON,
+            USE_EMOJI, PIPX_HOME_ALLOW_SPACE
```

Comment:
Keeping this list up to date seems tedious. Couldn't we import `pipx.commands.environment.environment_variables` and join via f-string, as done in other places?

Reply:
Good idea! Done in [3ab8d48](https://github.com/pypa/pipx/pull/1493/commits/3ab8d4893ae1e05ff465fb220e7c076db4078ec7)
I also used this approach to automatically test the presence of all documented environment variables in the output of `pipx environment`.